match/ouachita_da.py

```diff
@@ -7,41 +7,41 @@
 sys.path.append("../")
 
 
-def match_cprr_with_post(cprr, post):
-    dfa = cprr[["uid", "first_name", "last_name", "agency"]]
+def match_brady_with_post(brady, post):
+    dfa = brady[["uid", "first_name", "last_name"]]
     dfa.loc[:, "fc"] = dfa.first_name.fillna("").map(lambda x: x[:1])
+    dfa.loc[:, "lc"] = dfa.last_name.fillna("").map(lambda x: x[:1])
     dfa = dfa.drop_duplicates(subset=["uid"]).set_index("uid")
 
-    dfb = post[["uid", "first_name", "last_name", "agency"]]
+    dfb = post[["uid", "first_name", "last_name"]]
     dfb.loc[:, "fc"] = dfb.first_name.fillna("").map(lambda x: x[:1])
+    dfb.loc[:, "lc"] = dfb.last_name.fillna("").map(lambda x: x[:1])
     dfb = dfb.drop_duplicates(subset=["uid"]).set_index("uid")
 
     matcher = ThresholdMatcher(
-        ColumnsIndex(["fc", "agency"]),
+        ColumnsIndex(["fc", "lc"]),
         {
             "last_name": JaroWinklerSimilarity(),
             "first_name": JaroWinklerSimilarity(),
         },
         dfa,
         dfb,
     )
-    decision = 1
+    decision = 0
```

Comment:
This is a very low decision. Keep in mind that score below 0.5 is considered more likely to be different records.